cfme/containers/provider/openshift.py

```diff
@@ -30,7 +30,7 @@ def create(self, validate_credentials=True, **kwargs):
         # probably a js wait issue, not reproducible manually
         super(OpenshiftProvider, self).create(validate_credentials=validate_credentials, **kwargs)
 
-    def _href(self):
+    def href(self):
```

Comment:
Oh I don't see a problem with it - its just that it wasn't actually 'protected' by python. Wanted to make sure there wasn't a secondary intent here, something else I was missing.